doc/users/cmake_dealii.html

```diff
@@ -429,18 +429,6 @@ <h3>Feature configuration</h3>
       by <code>cmake</code> variables that may be set to either <code>ON</code>
       or <code>OFF</code>.
 
-      By default, <acronym>deal.II</acronym>'s <code>CMake</code> build scripts
-      will attempt to find the newest <code>C++</code> version available by
-      checking which language version flags work with the detected compiler. For
-      example: if <code>CMake</code> detects <code>GCC 5.1</code> or <code>Clang
-      3.6</code>, <acronym>deal.II</acronym> will be compiled
-      in <code>C++14</code> mode unless <code>C++14</code> support is explicitly
-      disabled. Another example: if <code>DEAL_II_WITH_CXX14=ON</code> is passed
-      to <code>CMake</code>, then <acronym>deal.II</acronym> will check if the
-      compiler supports enough of the <code>C++14</code> standard to enable
-      the <code>C++14</code>-specific parts of the library. The configuration
-      process will fail if a feature flag is passed in that cannot be supported.
```

Comment:
Can you add some information about setting the compiler flag to `-std=c++17` in case a user desires to use the newest features, which enable deal.II to provide this sugar out of the box? The text could be similar to what you wrote in the incompatibilities text above.

Reply:
I wrote a comment.